docs/wiki/blocks/block-body.md

```diff
@@ -0,0 +1,70 @@
+# Block Body
+
+General format of a **Block Body** in a Mimblewimble/Grin is basically a Mimblewimble transaction which consists of a list of **Inputs**, **Outputs** and **Kernels**.
```

Comment:
See above: Block -> Transaction

Reply:
Updated, but file will be moved and renamed at the last them once the review is concluded just to avoid mess caused by changing the URL of the rendered text.